modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/beanValidation.mustache

```diff
@@ -0,0 +1,53 @@
+{{#required}}
```

Comment:
Can beanValidation.mustache also bring in `{{>beanValidationParams}}`?  There looks like there's more common ground here...